hw2/03-count.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>03 - Count</title>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+  <script defer src="03-count.js"></script>
+</head>
+
+<body class="container-xs bg-primary d-flex justify-content-center align-items-center vw-100 vh-100">
+  <main class="bg-light rounded p-4 h-75 w-75">
```

Comment:
From my previous comment, I would recommend centering the main class by setting a width, then using margin auto. Reduces the amount of bootstrap classes needed! However, your approach works just as fine, just wanted to point this out!